src/install.ts

```diff
@@ -16,17 +17,21 @@ export default install;
  * @param release Release of MATLAB to be set up (e.g., "latest" or "R2020a").
  */
 export async function install(platform: string, release: string) {
-    // Install runtime system dependencies for MATLAB
-    await core.group("Preparing system for MATLAB", () =>
-        script.downloadAndRunScript(platform, properties.matlabDepsUrl, [release])
-    );
+    // Install runtime system dependencies for MATLAB on Linux
+    if (platform === "linux") {
+        await core.group("Preparing system for MATLAB", () =>
+            script.downloadAndRunScript(platform, properties.matlabDepsUrl, [release])
+        );
+    }
 
     // Invoke ephemeral installer to setup a MATLAB on the runner
     await core.group("Setting up MATLAB", () =>
-        script.downloadAndRunScript(platform, properties.ephemeralInstallerUrl, [
-            "--release",
-            release,
-        ])
+        script
+            .downloadAndRunScript(platform, properties.ephemeralInstallerUrl, [
+                "--release",
+                release,
+            ])
+            .then(ematlab.addToPath)
```

Comment:
why not the following?

```suggestion
        await script.downloadAndRunScript(platform, properties.ephemeralInstallerUrl, [
                "--release",
                release,
            ]);
            ematlab.addToPath();
```

Reply:
It needs to return a `Promise<void>`. If we add a return before `ematlab.addToPath()` and put curlies around the whole thing, that could work. I think the chaining is a bit more idiomatic though.